scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6151,26 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        parameters = _check_fit_input_parameters(self, data, args, kwds)
+        data, fshape, floc, fscale = parameters
+        if floc is None:
+            raise FitDataError(lower=0, distr="pareto", upper=np.inf,
```

Comment:
Why pass those first arguments into `FitDataError`? They're not used, right?

Reply:
Otherwise, I think this `fit` override looks great. Nice and clean, and hopefully the rest will be very simple like that with all the work that went into `_check_fit_input_parameters`.

Reply:
>  Why pass those first arguments into FitDataError? They're not used, right?

They are positional required arguments–but I could default them each to `None`.